poet/poet.py

```diff
@@ -94,22 +94,35 @@ class PackageMetadata:
     homepage: str
     url: str
     checksum: str
-    checksum_type: str
+    checksum_type: str = "sha256"
```

Comment:
i think line 150's edit was meant to be here
```suggestion
    checksum_type: str = field(default="sha256", init=False)
```
also you will have to add `field` to the things you import from `dataclasses`